### PR TITLE
New plugin : role-summary

### PR DESCRIPTION
**Pull request by nlamirault**
**Checklist for plugin developers:**

- [x] Read the [Plugin Naming Guide](https://sigs.k8s.io/krew/docs/NAMING_GUIDE.md) (for new plugins)
- [x] Verify the installation from URL or a local archive works (`kubectl krew install --manifest=[...] --archive=[...]`)

### CODE REVIEW

**Comment by nlamirault** [comment]
Ex:

```
$ krew install bindrole                          
Updated the local copy of plugin index.
Installing plugin: bindrole
Installed plugin: bindrole

kubectl bindrole --help                     
Usage of /home/nicolas/.krew/bin/kubectl-bindrole:
      --as string                      Username to impersonate for the operation
      --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
      --cache-dir string               Default HTTP cache directory (default "/home/nicolas/.kube/http-cache")
      --certificate-authority string   Path to a cert file for the certificate authority
      --client-certificate string      Path to a client certificate file for TLS
      --client-key string              Path to a client key file for TLS
      --cluster string                 The name of the kubeconfig cluster to use
      --context string                 The name of the kubeconfig context to use
      --insecure-skip-tls-verify       If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
      --kubeconfig string              Path to the kubeconfig file to use for CLI requests.
  -n, --namespace string               If present, the namespace scope for this CLI request
      --request-timeout string         The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests. (default "0")
  -s, --server string                  The address and port of the Kubernetes API server
  -k, --subject-kind string            The Kind of subject which is bound Roles. (default "ServiceAccount")
      --token string                   Bearer token for authentication to the API server
      --user string                    The name of the kubeconfig user to use
  -v, --version                        Print command version
pflag: help requested
```

File: plugins/bindrole.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: bindrole
+spec:
+  shortDescription: Finding Kubernetes Roles bound to a specified ServiceAccount, Group or User
```
Comment (corneliusweig) [inline plugins/bindrole.yaml]:
```suggestion
  shortDescription: Show a summary of rbac roles tied to a given ServiceAccount, Group, or User
```

**Comment by ahmetb** [comment]
Thanks for submitting your plugin @nlamirault.

Like @corneliusweig I'm a bit puzzled by the naming.

Description says it's a tool that "finds" bindings. But the name says "bind role" which is a _verb_ of assigning a role to someone. If the tool isn't doing that, I'll request a name change to something more appropriate.

We have plugins like `access-matrix`, `rbac-lookup`, `rbac-view` and `who-can` that more clearly explain what they do from their name. I prefer this plugin follows suit.

**Comment by ahmetb** [comment]
/hold
due to naming

**Comment by corneliusweig** [comment]
Hey @nlamirault, what's your status here? Are you still planning to have this included in krew-index?

**Comment by nlamirault** [comment]
@corneliusweig still on vacation. asap :)

**Comment by nlamirault** [comment]
> Thanks for submitting your plugin @nlamirault.
> 
> Like @corneliusweig I'm a bit puzzled by the naming.
> 
> Description says it's a tool that "finds" bindings. But the name says "bind role" which is a _verb_ of assigning a role to someone. If the tool isn't doing that, I'll request a name change to something more appropriate.
> 
> We have plugins like `access-matrix`, `rbac-lookup`, `rbac-view` and `who-can` that more clearly explain what they do from their name. I prefer this plugin follows suit.

**Comment by ahmetb** [comment]
Ping, are you still willing to pursue this?

**Comment by nlamirault** [comment]
@ahmetb i just renamed the plugin filename. Description was updated, name of the plugin too.

File: plugins/role-summary.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: role-summary
+spec:
+  shortDescription: Show a summary of rbac roles tied to a given ServiceAccount, Group, or User
```
Comment (corneliusweig) [inline plugins/role-summary.yaml]: Can you think of a shorter summary? This gets truncated at ~50 characters.

**Comment by ahmetb** [comment]
/close
Please let me know if you're still interested in pursuing, then I can reopen.

**Pull request closed without merging**